cfme/web_ui/menu.py

```diff
@@ -298,7 +298,7 @@ def sections(self):
                         ('containers_pods', 'Pods'),
                         ('containers_routes', 'Routes'),
                         ('containers_replicators', 'Replicators'),
-                        ('containers_services', 'Services'),
+                        ('containers_services', 'Container Services'),
```

Comment:
Well, probably not - as I think menu is split now into 5.5 and 5.6 as two different sections.